oneshot.d/browser-move-data-to-new-location

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# Moves Browser and Captive portal profiles to new location
+OLD_BROWSER_PROFILE_PATH="$HOME/.mozilla/mozembed"
+NEW_BROWSER_PROFILE_PATH="$HOME/.local/share/org.sailfishos/sailfish-browser/.mozilla"
+
+OLD_CAPTIVE_PORTAL_PROFILE_PATH="$HOME/.mozilla/captiveportal"
+NEW_CAPTIVE_PORTAL_PROFILE_PATH="$HOME/.local/share/org.sailfishos/sailfish-browser-captiveportal/.mozilla"
+
+if [ -e "$OLD_BROWSER_PROFILE_PATH/" ] && [ ! -e "$NEW_BROWSER_PROFILE_PATH/" ]
+then
+    rm -f $OLD_BROWSER_PROFILE_PATH/.parentlock
+    mkdir -p "$NEW_BROWSER_PROFILE_PATH"
+    if mv "$OLD_BROWSER_PROFILE_PATH/"* "$NEW_BROWSER_PROFILE_PATH/"
+    then
+        rmdir "$OLD_BROWSER_PROFILE_PATH"
+        rmdir "$HOME/.mozilla" > /dev/null 2>&1
+    fi
+fi
+
+if [ -e "$OLD_CAPTIVE_PORTAL_PROFILE_PATH/" ] && [ ! -e "$NEW_CAPTIVE_PORTAL_PROFILE_PATH/" ]
+then
+    rm -f $OLD_CAPTIVE_PORTAL_PROFILE_PATH/.parentlock
+    mkdir -p "$NEW_CAPTIVE_PORTAL_PROFILE_PATH"
+    if mv "$OLD_CAPTIVE_PORTAL_PROFILE_PATH/"* "$NEW_CAPTIVE_PORTAL_PROFILE_PATH/"
+    then
+        rmdir "$OLD_CAPTIVE_PORTAL_PROFILE_PATH"
+        rmdir "$HOME/.mozilla" > /dev/null 2>&1
```

Comment:
One more thing about this: This oneshot fails because this remove fgals if ~/.mozilla/extensions is still there and this is the last command executed so the exit status of the script is whatever this command exits with. On second run (== usually next reboot) this oneshot doesn't do anything and succeeds. That might be something to consider still. Otherwise it seems to work fine.

Reply:
We could do this at the end with rm -rf :)

Yeah, that extensions dir is not nice really. Maybe we just leave the .mozilla there for now and handle final cleanup in JB#52230

Reply:
> Maybe we just leave the .mozilla there for now and handle final cleanup in JB#52230

Sounds good to me.